tests/unit/models/transactions/test_did_set.py

```diff
@@ -62,3 +62,46 @@ def test_empty(self):
                 error.exception.args[0],
                 "{'did_set': 'Must have one of `did_document`, `data`, and `uri`.'}",
             )
+
+    def test_all_empty_fields(self):
+        with self.assertRaises(XRPLModelException):
+            DIDSet(account=_ACCOUNT, data="", did_document="", uri="")
+
+    def test_delete_multiple_fields(self):
+        # create a valid DID object
+        tx = DIDSet(
+            account=_ACCOUNT,
+            did_document=_VALID_FIELD,
+            uri=_VALID_FIELD,
+            data=_VALID_FIELD,
+        )
+        self.assertTrue(tx.is_valid())
+
+        # delete data field
+        tx = DIDSet(
+            account=_ACCOUNT,
+            data="",
+        )
+
+        self.assertTrue(tx.is_valid())
+
+        # delete URI field
+        tx = DIDSet(
+            account=_ACCOUNT,
+            uri="",
+        )
+
+        self.assertTrue(tx.is_valid())
+
+        # delete did_document field
+
+        # Note: It is invalid to render a DID LedgerObject empty by removing all the
+        # fields. However, that condition can only be detected by reading the
+        # LedgerEntry, which is out-of-scope for the library. rippled generated a
+        # `tecEMPTY_DID` error code in this case
```

Comment:
This is also unnecessary since the transaction is never actually submitted.

Reply:
Yeah, but I wanted to exercise the `_get_errors` function in the `DIDSet` model. Otherwise, we will probably need to do the same with an integration test

Reply:
`_get_errors` runs on model creation, not on transaction submission. I was referring to the comment, not the transaction, since it's not actually submitting the transaction.

Reply:
I've removed the comment and the test in [ed1c57c](https://github.com/XRPLF/xrpl-py/pull/717/commits/ed1c57c3e995b02cceb4b70a5e587dc7cfae3c52)